pages/[[server]]/@[account]/[status].vue

```diff
@@ -28,28 +26,24 @@ const { data: context, pending: pendingContext, refresh: refreshContext } = useA
   { watch: [isHydrated], immediate: isHydrated.value, lazy: true, default: () => shallowRef() },
 )
 
-const replyDraft = $computed(() => status.value ? getReplyDraft(status.value) : null)
+if (pendingContext)
+  watchOnce(pendingContext, scrollTo)
+
+if (pending)
+  watchOnce(pending, scrollTo)
+
+async function scrollTo() {
+  await nextTick()
 
-function scrollTo() {
   const statusElement = unrefElement(main)
   if (!statusElement)
     return
 
   statusElement.scrollIntoView(true)
 }
 
-onMounted(scrollTo)
-
-if (pendingContext) {
-  watchOnce(pendingContext, async () => {
-    await nextTick()
-    scrollTo()
-  })
-}
-
-const focusEditor = () => {
-  publishWidget.value?.focusEditor?.()
-}
+const publishWidget = ref()
```

Comment:
Convention suggest to use ref(null)

```suggestion
const publishWidget = ref(null)
```